build/pipelines/release.yml

```diff
@@ -1,48 +1,492 @@
+# This build should never run as CI or against a pull request.
 trigger: none
 pr: none
 
+pool:
+  name: Package ES Standard Build
+
+parameters:
+  - name: branding
+    displayName: "Branding (Build Type)"
+    type: string
+    default: Release
+    values:
+      - Release
+      - Preview
+  - name: buildTerminal
+    displayName: "Build Windows Terminal MSIX"
+    type: boolean
+    default: true
+  - name: buildTerminalVPack
+    displayName: "Build Windows Terminal VPack"
+    type: boolean
+    default: false
+  - name: buildWPF
+    displayName: "Build Terminal WPF Control"
+    type: boolean
+    default: false
+  - name: pgoBuildMode
+    displayName: "PGO Build Mode"
+    type: string
+    default: Optimize
+    values:
+      - Optimize
+      - Instrument
+      - None
+
+  - name: buildConfigurations
+    type: object
+    default:
+      - Release
+  - name: buildPlatforms
+    type: object
+    default:
+      - x64
+      - x86
+      - arm64
+
 variables:
-  baseYearForVersioning: 2019 # Used by build-console-int
-  versionMajor: 0
-  versionMinor: 1
-
-# When we move off PackageES for Versioning, we'll need to switch
-# name to this format. For now, though, we need to use DayOfYear.Rev
-# to unique our builds, as mandated by PackageES's Setup task.
-# name: '$(versionMajor).$(versionMinor).$(DayOfYear)$(Rev:r).0'
-#
-# Build name/version number above must end with .0 to make the
-# store publication machinery happy.
-name: 'Terminal_$(date:yyMM).$(date:dd)$(rev:rrr)'
-
-# Build Arguments:
-#   WindowsTerminalOfficialBuild=[true,false]
-#     true - this is running on our build agent
-#     false - running locally
-#   WindowsTerminalBranding=[Dev,Preview,Release]
-#     <none>  - Development build resources (default)
-#     Preview - Preview build resources
-#     Release - regular build resources
+  TerminalInternalPackageVersion: "0.0.7"
+
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+resources:
+  repositories:
+  - repository: self
+    type: git

```

Comment:
We should log a follow up on shallow checkout...

Reply:
Done!